From: Patrick Wang <patrick.wang.shcn@gmail.com>
Subject: selftests: vm: add the "settings" file with timeout variable
Date: Sat, 21 May 2022 16:38:25 +0800

The default "timeout" for one kselftest is 45 seconds, while some cases in
run_vmtests.sh require more time.  This will cause testing timeout like:

  not ok 4 selftests: vm: run_vmtests.sh # TIMEOUT 45 seconds

Therefore, add the "settings" file with timeout variable so users can set
the "timeout" value.

Link: https://lkml.kernel.org/r/20220521083825.319654-4-patrick.wang.shcn@gmail.com
Signed-off-by: Patrick Wang <patrick.wang.shcn@gmail.com>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/vm/settings |    1 +
 1 file changed, 1 insertion(+)

--- /dev/null
+++ a/tools/testing/selftests/vm/settings
@@ -0,0 +1 @@
+timeout=45
_
